### PR TITLE
report neighbour table sums

### PR DESCRIPTION
**Pull request by mweinelt**
Signed-off-by: Martin Weinelt <martin@darmstadt.freifunk.net>

### CODE REVIEW

File: update-telemetry.py [hunk 1]
```diff
@@ -190,6 +213,15 @@ def main():
             for key, value in zip(headings, values):
                 update['ipv4.%s.%s' % (section, key)] = value
 
+    for af in socket.AF_INET, socket.AF_INET6:
```
Comment (andir) [inline update-telemetry.py]: Could you make this a list or a tuple? Would be me obvious IMHO.

File: update-telemetry.py [hunk 2]
```diff
@@ -190,6 +213,15 @@ def main():
...
+        if af is socket.AF_INET:
```
Comment (andir) [inline update-telemetry.py]: when using a list of families for the loop you could just do:

```python
for af, prefix in [(socket.AF_INET, 'ipv4.Neigh'), (socket.AF_INET6, 'ipv6.Neigh')]:
    pass
```

**Review by andir — CHANGES_REQUESTED** [review]

File: update-telemetry.py (hunk 1 above)
Comment (mweinelt) [reply update-telemetry.py]: Technically this is a tuple.

File: update-telemetry.py (hunk 2 above)
Comment (mweinelt) [reply update-telemetry.py]: Updated.

**Pull request merged**